masu/database/report_db_cleaner.py

```diff
@@ -55,15 +56,23 @@ def purge_expired_report_data(self, expired_date):
             for line_item in line_item_query.all():
                 LOG.debug('Attempting to remove cost item data for bill id: %s, usage_start: %s',
                           bill_id, line_item.usage_start)
-                self._accessor.session.delete(line_item)
+                if simulate is False:
```

Comment:
This can just be `if simulate:`, which is the recommended PEP8 style.

For cases where a value defaults to `None`, it makes sense to check for `if foo_var is not None:` rather than the more general `if foo_var:` because various objects will also evaluate to `False` and it can make sense to see if the value differs from `None`.  

But, in this case, we're checking a boolean value already, which means any value that evaluates to `False` is acceptable. So, we can simplify the conditional.

Reply:
This whole function is going to go through some big changes once I do the performance optimizations. I'll definitely keep this in mind when I settle on the final version.